app/src/main/java/ch/epfl/skysync/models/reports/PilotReport.kt

```diff
@@ -0,0 +1,22 @@
+package ch.epfl.skysync.models.reports
+
+import ch.epfl.skysync.models.flight.Vehicle
+import ch.epfl.skysync.models.location.Location
+import ch.epfl.skysync.models.user.Pilot
+import java.sql.Time
+
+data class PilotReport(
+    val id: String,
```

Comment:
Here you can pass a default value `val id: String = UNSET_ID` so that we can create an instance without having to specify an ID

Reply:
Done